spark/src/main/java/org/apache/iceberg/spark/source/IcebergBatchScan.java

```diff
@@ -289,9 +272,13 @@ public String toString() {
     private transient Schema tableSchema = null;
     private transient Schema expectedSchema = null;
 
-    private ReadTask(
-        CombinedScanTask task, String tableSchemaString, String expectedSchemaString, FileIO fileIo,
-        EncryptionManager encryptionManager, boolean caseSensitive) {
+    BatchReadInputPartition(
+        CombinedScanTask task,
```

Comment:
Similar to above, the right style is to keep arguments on as few lines as possible.